### ISSUE TITLE
Sanity Start localhost not letting me signed in

### ISSUE THREAD

**Issue by RajrupDasid** [sanity-io/sanity#3108]
If you find a security vulnerability, do NOT open an issue. Email [security@sanity.io](mailto:security@sanity.io) instead.

**Describe the bug**
The bug is that , I sanity studio not letting me sign in on local host port 3333

When i type sanity start it's not letting me login unfortunately 

first show connect to dev-server 
then after choosing a provider that can be any not letting me log in 

when i click on login using any provider  it showing disconnect from dev-server 
then process for a bit then again redirect me back to main login screen 

well I am using Linux , and firefox developer edition browser and firefox main browser for doing such

**To Reproduce**

1. Go to 'terminal'
2. Type  'sanity start.'
3. open port on 3333 'on a browser'

**Expected behavior**
It should have to let me log in.

**Comment by rlpennell** [comment]
It's likely that your browser is blocking our login cookie. Can you try enabling third party cookies for the Studio's page to see if that resolves the issue?

**Comment by RajrupDasid** [comment]
> It's likely that your browser is blocking our login cookie. Can you try enabling third party cookies for the Studio's page to see if that resolves the issue?

yes that did work
but that is not a solution 
please fix this issue

**Comment by geoffreyjball** [comment]
Thanks for the feedback, Rajrup! We are working on a solution. I'm happy to hear Racheal's suggestion got you unblocked and able to use the studio. In the meantime, I'll mark this closed as a duplicate of #2198.

**Comment by alexhwoods** [comment]
Same

**Comment by ktor-vi** [comment]
Same

**Comment by akhtarali007** [comment]
same

> Hi guys - I am having the same issue however I get no error when looking at the console and it is not being solved by changing browser/cookies

same
